build_tools/scripts/bazel_to_cmake.py

```diff
@@ -167,7 +226,9 @@ def _convert_deps_block(self, **kwargs):
     #    package2::target
     deps = kwargs.get("deps")
     deps_list = [self._convert_target(dep) for dep in deps]
-    deps_list = sorted(list(set(deps_list)))  # Remove duplicates and sort.
+    #deps_list = sorted(list(set(deps_list)))  # Remove duplicates and sort.
+    # Remove `None` values and duplicates, preserving the original ordering.
```

Comment:
```suggestion
    # Remove Falsey (None and empty string) values and duplicates, preserving the original ordering.
```

I don't think we're actually getting `None` from anywhere anymore. I think this should be explicit that it's removing empty string as well.

Reply:
Yep, that was prior to replacing `return` with `return ""`. Thanks for catching this.